pygments/lexers/zig.py

```diff
@@ -122,8 +122,3 @@ class ZigLexer(RegexLexer):
             (r'"', String, '#pop')
         ]
     }
-
-    def get_tokens_unprocessed(self, text):
-        for index, token, value in \
-                RegexLexer.get_tokens_unprocessed(self, text):
-            yield index, token, value
```

Comment:
Why was this removed wholesale? I presume because that's equivalent to the parent class implementation?

Reply:
Since the parent class is RegexLexer, AFACIS it just delegates everything to it.